### PR TITLE
add pr template

### PR DESCRIPTION
**Pull request by calvin0327**
Signed-off-by: calvin0327 <wen.chen@daocloud.io>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add the template of pull request. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: scripts/helm-docs.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (Iceber) [inline scripts/helm-docs.sh:1]: 我们后续可以再增加一个校验 helm-docs 的 github action

Comment (calvin0327) [reply scripts/helm-docs.sh:1]: 体验了下，这个 helm-docs 比较实用

File: charts/clusterpedia/README.md.gotmpl [hunk 2]
```diff
@@ -0,0 +1,215 @@
+# Clusterpedia
```
Comment (Iceber) [inline charts/clusterpedia/README.md.gotmpl:1]: @calvin0327 readme 发生了一些修改，需要更新

https://github.com/clusterpedia-io/clusterpedia-helm/pull/12/files#diff-aa5f8d1db1ddd67c52ba91443f58f5e10ab5e3fd38101debae3b9ebcf02abb55

Comment (calvin0327) [reply charts/clusterpedia/README.md.gotmpl:1]: ok

Comment (calvin0327) [reply charts/clusterpedia/README.md.gotmpl:1]: done, we should update the  file`README.md.gotmpl` and then genete README.md via `./scripts/helm-docs.sh`.

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**